codegen.yml

```
@@ -1,6 +1,6 @@
 overwrite: true
 schema:
-  - "${BASE_API_URL}/graphql":
+  - "graphql.schema.json":
```

```suggestion
  - "${VEGA_SCHEMA_PATH}":
```
и скрипт, должно быть, не нужен

---

скрипт нужен, чтобы саму схему в репу положить

---

чтобы наш ci работал

---

так она же при кодгене сама появится

---

в нашем CI переменная окружения будет указывать на файл в репе, на стенде в YC тоже, а локально мы можем указать url под конкретный стенд